pkg/metapage/multi.go

```diff
@@ -105,7 +105,7 @@ func (m *LinkedMetaPage) Metadata() ([]byte, error) {
 	if _, err := m.rws.Seek(int64(m.rootMemoryPointerPageOffset())+24, io.SeekStart); err != nil {
 		return nil, err
 	}
-	buf := make([]byte, 4+m.rws.SlotSize())
+	buf := make([]byte, 1+m.rws.SlotSize())
```

Comment:
The test cases above passes regardless of what this number is set to. Do you have a suggestion for a breaking test case? A bit confused why we used to pad it with 4

Reply:
Ahh this is only the read buffer. Actually a value of `4` here would be fine, it will just overallocate the buffer by a bit. It will get truncated on line 114.

Reply:
In other words, I don't think you can create a test case that actually checks this.